### PR TITLE
DIT 229 View Application Details

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/application_details.scala.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                        <th scope="col"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @for(a <- c.attachments) {
+                            <td>@Dates.format(a.timestamp)</td>
+                            <td>@a.mimeType</td>
+                            <td>
+                                @if(a.isImage) {
+                                    <img src="@a.url" style="max-width: 100px; max-height: 100px"/>
```
Comment (pllaurado) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/application_details.scala.html]: add the <a> image </a>

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/case_summary.scala.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h1 class="heading-large">#@c.reference - @c.application.getType() for @c.applic
             @case_nav("summary", c.reference)
         </div>
         <div class="column-three-quarters">
-            <h3 class="heading-medium">Summary</h3>
+            <h3 class="heading-medium" style="margin-top: 0">Summary</h3>
```
Comment (pllaurado) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/case_summary.scala.html]: Could we group the styles on the css? :)

Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/case_summary.scala.html]: Sorted

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/application_details.scala.html (hunk 1 above)
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/application_details.scala.html]: Sorted

**Pull request merged**